### PR TITLE
v19.2.0 release notes

### PR DESCRIPTION
**Pull request by jseldess**

### CODE REVIEW

File: releases/v19.2.0.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
...ingle-node cluster with replication disabled without the need for any extra commands. Alongside this change, using `cockroach start` without the `--join` flag is now deprecated and will be removed in a future release.<br><br>Next, the [`cockroach demo`](../v19.2/cockroach-demo.html) command, which starts a temporary, in-memory CockroachDB cluster with immediate access to an interactive SQL shell, now comes pre-loaded with the `movr` dataset and a temporary enterprise license that expires after an hour. This means users can try out [enterprise features](../v19.2/enterprise-licensing.html) quickly before requesting a license for use in a persistent cluster.<br><br>Also, the `--nodes` flag now lets you specify the number of in-memory nodes to create for the demo, with the new `--demo-locality` flag setting locality information for each node. Alternatively, the `--geo-partitioned-replicas` flag lets you start a 9-node demo cluster with the [Geo-Partitioned Replicas](../v19.2/topology-geo-partitioned-replicas.html) data topology applied automatically to the `movr` database.<br><br>Finally, the `--with-load` command lets you run `movr` as an active workload against the in-memory cluster.
```
Comment (knz) [inline releases/v19.2.0.md]: > Alongside this change, using `cockroach start` without the `--join` flag is now deprecated and will be removed in a future release.

This particular sentence is not really about "local testing". it's rather a general operational change.

File: releases/v19.2.0.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+[**Built-in Workload Improvements**](../v19.2/cockroach-workload.html) | Cockroach Lab's fictional vehicle-sharing app, [MovR](../v19.2/movr.html), is now available as a sample workload using the `cockroach workload` CLI command. Also, the `--display-every` and `--display-format` flags now make it possible to control the frequency and format for printing per-operation statistics for several workloads.
+[**Showing String Collations**](../v19.2/collate.html#show-collation-for-strings) | You can now use the `pg_collation_for` built-in function, or its alternative syntax form `COLLATION FOR`, to return the locale name of a collated string.
```
Comment (knz) [inline releases/v19.2.0.md]: This one is rather esoteric, I doubt it deserves a call-out.

File: releases/v19.2.0.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+[**Interactive SQL Shell Commands**](../v19.2/use-the-built-in-sql-client.html#commands) | Within the interactive SQL shell, the `\l`, `\dt`, `\d`, and `\d <table>` commands now list all databases in the CockroachDB cluster, show the tables of the current schema in the current database, list the users for all databases, and show details about columns in the specified table, respectively.
+[**Removing Manual Splits**](../v19.2/unsplit-at.html) | The new `UNSPLIT AT` statement removes a [split enforcement](../v19.2/split-at.html) on a key-value range, at a specified row in a table or index, allowing CockroachDB to [merge the range](../v19.2/range-merges.html) as needed.
+[**Locking and `FOR UPDATE`**](../v19.2/porting-postgres.html#locking-and-for-update) | Because CockroachDB only supports `SERIALIZABLE` isolation, locking is not required to order concurrent transactions relative to each other. However, the `FOR UPDATE` locking clause is now supported in selection queries for database migration compatibility only.
```
Comment (knz) [inline releases/v19.2.0.md]: > the `FOR UPDATE` locking clause is now supported in selection queries **as a no-op** ...

**Comment by jess-edwards** [comment]
@jseldess LGTM

**Comment by jess-edwards** [comment]
@jseldess this is the arrow for SIGN IN `&#8594;`

**Comment by bdarnell** [comment]
We also need to mention that the timestamp precision change (present in previous builds including rc4) has been reverted.

**Pull request merged**